lms/djangoapps/certificates/utils.py

```diff
@@ -250,3 +255,25 @@ def get_preferred_certificate_name(user):
         name_to_use = ''
 
     return name_to_use
+
+
+def get_certificate_configuration_from_credentials(course_id: CourseKey, mode: str) -> Optional[Response]:
+    """
+    Makes a request to the credentials service to get a certificate configuration of the course.
+    """
+    course_certificates_api_url = urljoin(f'{get_credentials_api_base_url()}/', 'course_certificates/')
+    api_url = urljoin(course_certificates_api_url, f'?course_id={course_id}&certificate_type={mode}')
+
+    try:
+        credentials_client = get_credentials_api_client(
+            User.objects.get(username=settings.CREDENTIALS_SERVICE_USERNAME)
+        )
+        response = credentials_client.get(api_url)
+    except Exception:   # pylint: disable=broad-except
+        log.warning(
+            'There is no certificate configuration in credentials for a course [%s] with a mod [%s]',
+            course_id,
+            mode
+        )
+    else:
+        return response
```

Comment:
Please answer me why u do not use the function `get_api_data`?

Reply:
Because I didn't know it existed.